src/reader.js

```diff
@@ -148,8 +148,12 @@ class Reader extends EventEmitter {
 
     // Trigger a query of the configured `lookupdHTTPAddresses`.
     const endpoint = this.roundrobinLookupd.next();
-    lookup(endpoint, this.topic, (err, nodes = []) =>
-      nodes.map(n => this.connectToNSQD(n.broadcast_address, n.tcp_port)));
+    lookup(endpoint, this.topic, (err, nodes = []) => {
+      if(err){
+        this.emit(Reader.ERROR, err);
+      }
```

Comment:
Hey, Could you read what I have written here: https://github.com/dudleycarr/nsqjs/pull/140/commits/867a5b84fccfb7364ec31fc02f8932d8f2d112a6#r119331938